.github/workflows/check-sdist.yml

```diff
@@ -19,15 +19,15 @@ jobs:
         # scipy and cython are required to build sdist
         run: |
           python -m pip install --upgrade pip
-          pip install check-manifest scipy cython
+          pip install check-sdist scipy cython
       - run: |
-          check-manifest -v
+          check-sdist
```

Comment:
Do you think it'll be useful to include `--inject-junk` here to?

Reply:
I am not sure but I included it :wink: